app/javascript/components/map/mixins/mixin-bounding-box.js

```diff
@@ -31,23 +39,41 @@ export default {
       }
     },
 
-    zoomTo (boundsUrl) {
+    zoomTo (options) {
       axiosGetWithoutCSRF(
-        boundsUrl.url, 
-        this.getExtentResponseZoomToHandler(boundsUrl.padding)
+        options.extent_url.url, 
+        this.getZoomAndPopupHandler(options)
       )
     },
 
-    getExtentResponseZoomToHandler (padding) {
+    getZoomAndPopupHandler (options) {
       return res => {
         const extent = res.data.extent
-  
-        if (extent) {
-          this.map.fitBounds(this.getBoundsFromExtent(extent, padding))
+
+        if (!extent) { return }
```

Comment:
Would ESLint complain if it were `if (!extent) return` instead?

Reply:
Probably not, but it is what we do in general and I think my preference, but only slightly so. I don't mind which particularly, so happy to change if you have stronger feelings than I do!

Reply:
I'm not too fussed either, I used to really not enjoy seeing inline ifs at all. 

I prefer the return to be dropped to its own line if you want to have braces, I like seeing both inline (but without braces) or not inline but on a new line